api/src/test/java/org/openmrs/util/OpenmrsUtilUnitTest.java

```diff
@@ -137,4 +147,128 @@ public void join_shouldReturnListElementsJoinedBySeparatorIfGivenSeparatorIsNull
 
 		assertThat(OpenmrsUtil.join(moduleList, null), is("module1nullmodule2"));
 	}
+
+	@Test
+	public void isStringInArray_shouldPassIfStringIsInArray() {
+
+		String[] array = new String[]{"element1","element2","element3"};
+
+		assertTrue(OpenmrsUtil.isStringInArray("element1", array));
+	}
+
+	@Test
+	public void isInNormalNumericRange_shouldNotAllowNullValuesAsLowerAndHigherNormal() {
+
+		ConceptNumeric concept = new ConceptNumeric();
+		concept.setHiNormal(null);
+
+		assertFalse(OpenmrsUtil.isInNormalNumericRange(5.67f, concept));
+	}
+
+	@Test
+	public void isInNormalNumericRange_shouldPassIfValueWithinRange() {
+
+		ConceptNumeric concept = new ConceptNumeric();
+		concept.setHiNormal(10.34d);
+		concept.setLowNormal(3.67d);
+
+		assertTrue(OpenmrsUtil.isInNormalNumericRange(5.67f, concept));
+	}
+
+	@Test
+	public void isInCriticalNumericRange_shouldNotAllowNullValuesAsLowerAndHigherCritical() {
```

Comment:
same as above please adapt the name and add test case for the other case `getLowCritical == null`